benchmark/perf-tests/common/image-preload-daemonset.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ray-image-preloader
```

Comment:
For 1K and 5K tests, a lot of time was spent on pulling Ray images so I added pre-loader daemonset that runs before the tests